accounts/forms.py

```diff
@@ -277,6 +277,15 @@ def __init__(self, *args, **kwargs):
         self.user = kwargs.pop('user', None)
         super(EmailResetForm, self).__init__(*args, **kwargs)
 
+    def clean_email(self):
+        email = self.cleaned_data["email"]
+        try:
+            User.objects.get(email__iexact=email)
+            raise forms.ValidationError(_("A user using that email address already exists."))
```

Comment:
As discussed in the office, for now we'll just log when this happens and we'll decide which further action to take. https://github.com/MTG/freesound/pull/763/commits/f2329fb67f040fcbb3e67f31765ffff399766ede